### PR TITLE
doc: minikube extension testing

### PR DESCRIPTION
**Pull request by amisskii**
issue: https://github.com/containers/podman-desktop-internal/issues/135

### CODE REVIEW

File: docs/scenarios/minikube/minikube.md [hunk 1]
```diff
@@ -0,0 +1,99 @@
+# Minikube Extension Testing
+
+## Prerequisites
+
+Ensure the following are installed:
+
+- **Podman** or **Docker**
+- **Virtualization Software** for Windows/Mac (e.g., WSL, Hyper-V, Libkrun, AppleHV), with minimum requirements:
+  - **CPU**: 2 cores
+  - **Memory**: 2048 MB
```
Comment (odockal) [inline docs/scenarios/minikube/minikube.md:10]: I do no think minikube will run with only 2 GB of memory properly. Is there some minimal requirements for minikube?

File: docs/scenarios/minikube/minikube.md [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+1. Open the **Podman Desktop** and navigate to the **Extensions** page.
+2. **Install the Minikube extension** from an OCI image.
+3. **Verify Installation**: Check that the Minikube extension appears under the installed extensions tab on the Extensions page.
+
+4. **Install Minikube CLI**:
+   - Go to **Settings > CLI Tools**.
+   - Find the **Minikube** tab and click the **Install** button. Choose the desired CLI version.
+
+5. **Verify CLI Installation**:
+   - Navigate to **Settings > Resources** and confirm that the **Minikube** tab is visible. This confirms that the extension is active and the CLI is installed.
```
Comment (odockal) [inline docs/scenarios/minikube/minikube.md]: tab or card.

File: docs/scenarios/minikube/minikube.md (hunk 1 above)
Comment (amisskii) [reply docs/scenarios/minikube/minikube.md:10]: Yes: 
- **Virtualization Software** for Windows/Mac (e.g., WSL, Hyper-V, Libkrun, AppleHV), **with minimum requirements**:
  - **CPU**: 2 cores
  - **Memory**: 2048 MB

Comment (odockal) [reply docs/scenarios/minikube/minikube.md:10]: oh, nice. Ok then.

**Review by odockal — APPROVED** [review]

**Pull request merged**